bids2openminds/main.py

```diff
@@ -132,7 +132,10 @@ def subjects_creation(subject_id, layout_df, layout):
         state_cash = []
         for sesion in sessions:
```

Comment:
can we fix the typo here `for session in sessions` ?

Reply:
leave it to the next PR if too much refactoring is required

Reply:
It should be a fairly simple PR, but I think we should do it in another pull request, as it is not related to this one.